docs/DeveloperGuide.md

```diff
@@ -382,6 +382,28 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 2. GuestList shows an exit message
 3. GuestList exits the program
 
+**Use case: Get the check-in/out date of a guest**
```

Comment:
Should we change the Use Case our program sounds more like it can list out the details of a particular guest? So we cannot only purely get the check-in/out date,